### PR TITLE
ci: fix adding issue to board github action task

### PR DESCRIPTION
**Pull request by ggera**
## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/sdk-incompatible-with-docs.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 ---
 
-- [ ] Check if any docs needs to be updated, and if needed do so
+- [ ] Click on link below to investigate logs of failing tests
```
Comment (ntn-x2) [inline .github/ISSUE_TEMPLATE/sdk-incompatible-with-docs.md]: What's the reason for this change?

Comment (ggera) [reply .github/ISSUE_TEMPLATE/sdk-incompatible-with-docs.md]: Undone
